### PR TITLE
doc set up 

### PR DESCRIPTION
**Pull request by arejula27**
I modify de readme.md for explaining installation and a basic usage

### CODE REVIEW

**Review by fjlopez — CHANGES_REQUESTED** [review]
Move the changes to the file **description.md**

File: README.md [hunk 1]
```diff
@@ -15,3 +15,22 @@ User name | NIA | Travis-CI|Score
 ----------|-----|----------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml/badge.svg)](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race/actions/workflows/ci.yml)
 your name | your nia | your Travis-ci status
+
+## Set up
+For building and running the app you will need the next resources:
+- Kotlin, you can get it following the next [web](https://kotlinlang.org/docs/command-line.html)
+- Java11, you can easily download it in the official oracle web
```
Comment (fjlopez) [inline README.md:22]: There are alternatives such as https://adoptopenjdk.net/

File: README.md [hunk 2]
```diff
@@ -15,3 +15,22 @@ User name | NIA | Travis-CI|Score
...
+For building it you only have to use the tool **gradelw**, for listing all the options 
+use the tasks command. 
+
+On the list you will see two important commands: **build** and **bootRun**, the first for building and the other for running the aplication.
+
+For checking if it works you can find the next page on localhost:8080
+![image](https://user-images.githubusercontent.com/46299278/133663363-de39b281-131e-4c4e-8e35-e32b4c5c18a5.png)
+
+## Testing the app
+for testing all the unitary test run de command ** gradelw check**
```
Comment (fjlopez) [inline README.md:35]: Fix the typo in the command.

**Comment by fjlopez** [comment]
Some changes requested. Don't close this PR.

**Comment by arejula27** [comment]
I moved the explanation to the correct file and change the inline code typos.

**Pull request merged**